src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/PropertySymbolReferenceFinder.cs

```diff
@@ -183,6 +187,53 @@ private static bool IsForEachProperty(IPropertySymbol symbol)
             return locations.ToImmutableAndFree();
         }
 
+        private async Task<ImmutableArray<FinderLocation>> FindConditionalAccessReferencesAsync(
+            IPropertySymbol symbol, Document document, SemanticModel semanticModel,
+            FindReferencesSearchOptions options, CancellationToken cancellationToken)
+        {
+            if (options.AssociatePropertyReferencesWithSpecificAccessor)
+            {
+                // Looking for individual get/set references.  Don't find anything here. 
+                // these results will be provided by the PropertyAccessorSymbolReferenceFinder
+                return ImmutableArray<FinderLocation>.Empty;
+            }
+
+            var symbolsMatch = GetStandardSymbolsNodeMatchFunction(symbol, document.Project.Solution, cancellationToken);
+
+            var syntaxRoot = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+            var syntaxFacts = document.GetLanguageService<ISyntaxFactsService>();
+            var semanticFacts = document.GetLanguageService<ISemanticFactsService>();
+
+            var conditionalAccessExpressions = syntaxRoot.DescendantNodes().Where(syntaxFacts.IsConditionalAccessExpression);
+            var locations = ArrayBuilder<FinderLocation>.GetInstance();
+
+            foreach (var node in conditionalAccessExpressions)
+            {
+                cancellationToken.ThrowIfCancellationRequested();
+
+                var (matched, reason) = symbolsMatch(node, semanticModel);
+                if (matched)
+                {
+                    syntaxFacts.GetPartsOfConditionalAccessExpression(node, out var expression, out var argumentList);
+
+                    if (symbolsMatch(expression, semanticModel).matched)
+                    {
+                        // Element access with explicit member name (allowed in VB).
+                        // We have already added a reference location for the member name identifier, so skip this one.
```

Comment:
i don't really understand this comment.  can you add a test demonstrating when this would be hit?

Reply:
This comment was copy-pasted from already existing source code in this class. The duplication of the comment is cleaned up. I will take a look if I can find out more about the comment to demonstrate it/ make it easier to verify.